packages/EasyTest/src/Coverage/Locators/CoverageResolverLocator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyTest\Coverage\Locators;
+
+use EonX\EasyTest\Interfaces\CoverageResolverInterface;
+use EonX\EasyTest\Interfaces\CoverageResolverLocatorInterface;
+use InvalidArgumentException;
+use Symfony\Contracts\Service\ServiceProviderInterface;
+
+final class CoverageResolverLocator implements CoverageResolverLocatorInterface
+{
+    /**
+     * @var \Symfony\Contracts\Service\ServiceProviderInterface
+     */
+    private $serviceLocator;
+
+    public function __construct(ServiceProviderInterface $serviceLocator)
+    {
+        $this->serviceLocator = $serviceLocator;
+    }
+
+    public function getCoverageResolver(string $filePath): CoverageResolverInterface
+    {
+        $extension = pathinfo($filePath, PATHINFO_EXTENSION);
```

Comment:
Please namespace the `\pathinfo()` method.

Reply:
Please namespace the `\PATHINFO_EXTENSION` constant as well.

Reply:
done